.agents/skills/advantagekit-replay/scripts/compare_replay_topics.py

```
@@ -0,0 +1,146 @@
+def topic_names(repo: Path, log: Path) -> set[str]:
+    with tempfile.TemporaryDirectory(prefix="akit-replay-topics-") as tmp:
+        tmp_path = Path(tmp)
+        source = tmp_path / "ListTopicNames.java"
+        classes = tmp_path / "classes"
+        classes.mkdir()
+        source.write_text(JAVA_SOURCE)
+        root = wpilib_root()
+        version = wpilib_version(root)
+        java_home = root / "jdk"
+        wpiutil = root / f"maven/edu/wpi/first/wpiutil/wpiutil-java/{version}/wpiutil-java-{version}.jar"
+        subprocess.run([str(java_home / "bin/javac"), "-cp", str(wpiutil), "-d", str(classes), str(source)], cwd=repo, check=True)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)